site/content/en/development/github-actions.md/verify-code.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
could you place it under https://kubernetes-sigs.github.io/cloud-provider-azure/development/dependencies/?

Reply:
done